civihr_default_theme/scss/base/_custom.scss

```diff
@@ -278,22 +278,6 @@ body {
     }
 }
 
-/**
- * Customization of the standard Bootstrap button group
- */
-.btn-group {
```

Comment:
Removed this code as we are changing the look of Navs using ".nav" classes which makes this code unnecessary.